File: app/src/main/java/cc/smartcash/wallet/Models/UserRecoveryKey.java
package cc.smartcash.wallet.Models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonIgnoreProperties(ignoreUnknown = true)
public class UserRecoveryKey {

    @JsonProperty("recoveryKey")
    private String recoveryKey;
    @JsonProperty("termsVersion")
    private String termsVersion;
    @JsonProperty("termsText")
    private String termsText;

    @JsonProperty("recoveryKey")
    public String getRecoveryKey() {
        return recoveryKey;
    }

    @JsonProperty("recoveryKey")
    public void setRecoveryKey(String recoveryKey) {
        this.recoveryKey = recoveryKey;
    }

    @JsonProperty("termsVersion")
    public String getTermsVersion() {
        return termsVersion;
    }

    @JsonProperty("termsVersion")
    public void setTermsVersion(String termsVersion) {
        this.termsVersion = termsVersion;
    }

    @JsonProperty("termsText")
    public String getTermsText() {
        return termsText;
    }

    @JsonProperty("termsText")
    public void setTermsText(String termsText) {
        this.termsText = termsText;
    }

}

File: app/src/main/java/cc/smartcash/wallet/Models/UserRecoveryKey.kt
package cc.smartcash.wallet.Models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty
import java.io.Serializable

@JsonIgnoreProperties(ignoreUnknown = true)
data class UserRecoveryKey(
        @JsonProperty("recoveryKey")
        @get:JsonProperty("recoveryKey")
        @set:JsonProperty("recoveryKey")
        var recoveryKey: String? = null,

        @JsonProperty("termsVersion")
        @get:JsonProperty("termsVersion")
        @set:JsonProperty("termsVersion")
        var termsVersion: String? = null,

        @JsonProperty("termsText")
        @get:JsonProperty("termsText")
        @set:JsonProperty("termsText")
        var termsText: String? = null

) : Serializable
